app/assets/stylesheets/show.scss

```diff
@@ -133,10 +154,10 @@
     @extend .header;
   }
 
-  .caption-header > caption {
+  .caption-header>caption {
```

Comment:
I think this is harder to read without the spaces.

Reply:
This is my editor auto-fixing spaces. I'm happy to adjust.